internal/common/function/function.go

```diff
@@ -95,6 +95,15 @@ func ValidateBackupSpec(ctx context.Context, clientInstance client.Client, oadpN
 			return fmt.Errorf("NonAdminBackup spec.backupSpec.includedNamespaces can not contain namespaces other than: %s", nonAdminBackup.Namespace)
 		}
 	}
+
+	if nonAdminBackup.Spec.BackupSpec.IncludeClusterResources != nil && *nonAdminBackup.Spec.BackupSpec.IncludeClusterResources {
+		return fmt.Errorf("Non-admin users are not allowed to set includeClusterResources to true, it must be either nil or false")
```

Comment:
would change message here to be more user friendly (it may not know what `nil` means)

and would also be more precise: **includeClusterResources** -> **NonAdminBackup spec.backupSpec.includeClusterResources**

what about something like this
```
NonAdminBackup spec.backupSpec.includeClusterResources can not be set or must be set to false
```